File: mario-more/mario.c
#include<stdio.h>
#include<cs50.h>

int main(){
    int height;
    do{
        height = get_int("Enter height here :");
    }while (height < 1 || height > 8);

    for (int row = 0; row<height ; row++){
        for(int space = 0; space < height - row - 1; space ++){
            printf(" ");
        }

        for(int column = 0; column <= row; column ++){
            printf("#");
        }
        printf("  ");
        for(int column = 0; column <= row; column ++){
            printf("#");
        }
        printf("\n");
    }

}

File: sentimental-mario-more/mario.py
from cs50 import get_int
def main():
    height = 0
    while True:
        height = get_int(input("Enter height here: "))
        if height >= 1 and height <= 8:
            break

    for row in range(height):
        for space in range(height - row - 1):
            print(" ", end="")
        for column in range(row + 1):
            print("#", end="")
        print("  ", end="")
        for column in range(row + 1):
            print("#", end="")
        print()

if __name__ == "__main__":
    main()
